### PR TITLE
Man 183 arrange an appointment

### PR DESCRIPTION
**Pull request by neil-mills**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -34,15 +34,18 @@
...
-        ],
+        "import/prefer-default-export": 0,
+        "@typescript-eslint/no-explicit-any": 0,
+        // "@typescript-eslint/no-unused-vars": [
+        //   1,
+        //   {
+        //     "argsIgnorePattern": "res|next|^err|_",
+        //     "ignoreRestSiblings": true
+        //   }
+        // ],
```
Comment (pmcphee77) [inline .eslintrc.json:48]: Can we remove this or uncomment?

File: server/middleware/getSentences.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express'
+import { HmppsAuthClient } from '../data'
+import MasApiClient from '../data/masApiClient'
+
+export const getSentences = (hmppsAuthClient: HmppsAuthClient) => {
+  return async (req: Request, res: Response, next: NextFunction) => {
+    const number = (req?.query?.number as string) || ''
+    const crn = req.params.crn as string
+
+    // if (number || !req?.session?.data?.sentences?.[crn]) {
```
Comment (pmcphee77) [inline server/middleware/getSentences.ts:10]: Remove these comments?

File: server/middleware/getSentences.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    const masClient = new MasApiClient(token)
+    const allSentences = await masClient.getSentences(crn, number)
+    req.session.data = {
+      ...(req?.session?.data || {}),
+      sentences: {
+        ...(req?.session?.data?.sentences || {}),
+        [crn]: allSentences.sentences,
+      },
+    }
+    // }
```
Comment (pmcphee77) [inline server/middleware/getSentences.ts:21]: same here

File: server/middleware/setUpStaticResources.ts [hunk 4]
```diff
@@ -19,6 +19,7 @@ export default function setUpStaticResources(): Router {
     '/assets',
     '/assets/stylesheets',
     '/assets/js',
+    '/server/assets',
```
Comment (pmcphee77) [inline server/middleware/setUpStaticResources.ts:22]: Just checking - should these be copied over?

File: server/assets/application.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-restricted-globals */
```
Comment (pmcphee77) [inline server/assets/application.ts:1]: Is this the typescript for the javascript checked in above?

File: wiremock/mappings/X778160-full.json [hunk 6]
```diff
@@ -163,9 +163,8 @@
         "jsonBody": {
```
Comment (pmcphee77) [inline wiremock/mappings/X778160-full.json:163]: Can we remove the .DS_Store file that has been checked in?

**Review by pmcphee77 — CHANGES_REQUESTED** [review]

File: server/middleware/setUpStaticResources.ts (hunk 4 above)
Comment (neil-mills) [reply server/middleware/setUpStaticResources.ts:22]: Yes this should be removed. I need to work on migrating the application js file to ts.

File: .eslintrc.json (hunk 1 above)
Comment (neil-mills) [reply .eslintrc.json]: Resolved

File: server/middleware/getSentences.ts (hunk 3 above)
Comment (neil-mills) [reply server/middleware/getSentences.ts]: Resolved

File: wiremock/mappings/X778160-full.json (hunk 6 above)
Comment (neil-mills) [reply wiremock/mappings/X778160-full.json:163]: Resolved

File: server/middleware/getSentences.ts (hunk 2 above)
Comment (neil-mills) [reply server/middleware/getSentences.ts]: Resolved

**Review by pmcphee77 — APPROVED** [review]

**Pull request merged**